packages/metadata/test/unit/inspector.test.ts

```diff
@@ -631,6 +633,15 @@ describe('Inspector for design time metadata', () => {
     expect(meta).to.eql(MyClass);
   });
 
+  it('inspects design time type for properties with union type', () => {
```

Comment:
This looks like unrelated changes to me. @raymondfeng could you please check these changes were intended?